devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -88,27 +88,33 @@ private handleReportAttributeMessage(String description) {
     }
 }
 
-private handleBatteryEvent(rawValue) {
-    def linkText = getLinkText(device)
-
+/**
+ * Create battery event from reported battery voltage.
+ *
+ * @param volts Battery voltage in .1V increments
+ */
+private handleBatteryEvent(volts) {
     def eventMap = [
         name: 'battery',
         value: '--'
```

Comment:
I did it that way because that's the way the other sensors were implemented. Do you think I should change them all to not report any battery event if the value is 0 or 255?